config/database.yml

```diff
@@ -57,6 +57,7 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
+    #url: postgres://localhost:5432/early_years_foundation_recovery_test
```

Comment:
Non-docker leftovers 👀

Reply:
I need that so I can run non-docker yes.